modules/lead/sdm/variables.tf

```diff
@@ -38,11 +38,15 @@ variable "product_stack" {
   default = "product-local"
 }
 
-variable "product_vars" {
+variable "producttype_vars" {
```

Comment:
nitpick: technically these are product variables, but we're using the `ProductType` to help define them.  would you mind renaming this variable back to `product_vars`?

Reply:
Changed back to `product_vars`